packages/ref-metrics-pmp/tests/unit/test_metrics.py

```diff
@@ -0,0 +1,62 @@
+import pathlib
+
+import pytest
+from cmip_ref_metrics_example.example import GlobalMeanTimeseries, calculate_annual_mean_timeseries
+
+from cmip_ref_core.datasets import DatasetCollection, MetricDataset, SourceDatasetType
+from cmip_ref_core.metrics import MetricExecutionDefinition
+
+
+@pytest.fixture
+def metric_dataset(cmip6_data_catalog) -> MetricDataset:
+    selected_dataset = cmip6_data_catalog[
+        cmip6_data_catalog["instance_id"]
+        == "CMIP6.ScenarioMIP.CSIRO.ACCESS-ESM1-5.ssp126.r1i1p1f1.Amon.tas.gn.v20210318"
+    ]
+    return MetricDataset(
+        {
+            SourceDatasetType.CMIP6: DatasetCollection(
+                selected_dataset,
+                "instance_id",
+            )
+        }
+    )
+
+
+def test_annual_mean(sample_data_dir, metric_dataset):
+    annual_mean = calculate_annual_mean_timeseries(metric_dataset["cmip6"].path.to_list())
+
+    assert annual_mean.time.size == 286
```

Comment:
```suggestion
    assert annual_mean.time.size == 11
```

This will also be needed for when you merge the main branch into this branch as we have a new set of sample data which don't include as many timesteps, but are contiguous in time.